setup.py

```diff
@@ -8,8 +8,8 @@
 from setuptools import setup, find_packages
 from io import open
 
-MAJOR = 6
-MINOR = 2
+MAJOR = 7
+MINOR = 0
 MICRO = 0
 
 IS_RELEASED = False
```

Comment:
I don't know the release mechanism of traitsui all that well. Is this flag supposed to be changed to True later?

Reply:
Hmmm this branch is targeting master so it makes sense that flag is not changed in the diff... Resolving.

Reply:
We'll toggle this on the actual release commit, then bump the version to 7.1.0 unreleased as a second commit, then merge.

Reply:
Yep. Make sense. Thanks!